parm/atm/jcb-prototype_3dvar-fv3inc.yaml.j2

```diff
@@ -0,0 +1,10 @@
+# Algorithm
+# ---------
+algorithm: fv3jedi_fv3inc_variational
+
+# Testing things
```

Comment:
Similarly with these. Can we get away with only seeing all these things about testing in the JCB YAML files that live in the testing directory?